docs/pages/desktop-access/getting-started.mdx

```diff
@@ -25,78 +25,98 @@ This guide requires you to have:
   CS)](https://docs.microsoft.com/en-us/windows-server/networking/core-network-guide/cncg/server-certs/install-the-certification-authority)
   installed in the domain
 - Access to a Domain Controller
-- LDAP credentials for Active Directory (usually the same credentials you use
-  to log into the Domain Controller)
 - An existing Teleport cluster and user, version 8.0 or newer
-  - see [Teleport Getting Started](../getting-started.mdx) if you're new to Teleport
+  - See [Teleport Getting Started](../getting-started.mdx) if you're new to Teleport
 - A Linux server to run the Teleport Desktop Access service on
-  - you can reuse an existing server running any other Teleport instance
+  - You can reuse an existing server running any other Teleport instance
 
-## Step 1/3. Configure Teleport
+## Step 1/5: Create a User, Group, and Group Policy Object
+
+Teleport requires a service account to connect to your Active Directory domain. In theory any account with the required permissions can be used;
```

Comment:
Nit: if you remove: "In theory anymore  account with the ... ;" the sentence will read better.